**Exercício de Casa 🏠 Teste de Hipótese**

**Professora Débora**

**Cenário:**

Queremos avaliar se a altura média da turma on35 é significativamente diferente da altura média nacional da população feminina brasileira, que é de 160 cm. 

**Objetivo:**

    Verificar se a altura média da turma é significativamente diferente de 160 cm.

**Dados da Amostra:**

    Altura da turma (em cm).

1) Coletar a altura média de cada uma das alunas, excluindo as alturas da Professora, Facilitadora e Monitora.

**Amostra: 30 alturas coletadas**

(165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168, 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166, 160)






In [3]:
import numpy as np

#Criando uma váriavel altura, para calcular a altura média da turma on35
altura_media_turma_on35 = np.array([165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168, 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166, 160])

#Cálculo médio altura -turma on-35
media_altura = np.mean(altura_media_turma_on35)

#Imprimindo a média da altura
print(f"A altura média da turma on-35 é: {media_altura:.2f} anos")


A altura média da turma on-35 é: 163.60 anos



**2) Exercício: Teste z para Altura Média da Turma**

In [4]:
#Importando as bibliotecas 
#Coleta de dados de idade das estudantes da turma on35 - Amostra = 30 alturas
import numpy as np
from statsmodels.stats.weightstats import ztest

# Dados da amostra
altura_turma = np.array([165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168, 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166, 160])
media_nacional = 160

# Teste z para uma amostra
z_statistic, p_value = ztest(altura_turma, value=media_nacional)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}, p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: 3.5569, p-value: 0.0004



**Perguntas:**


**3) Calcule a estatística *z* e o valor *p* para verificar se a altura média da turma é significativamente diferente de 160 cm.**

**Resultados: Teste z para uma amostra:**

**z-statistic: 3.5569**

**p-value: 0.0004**


O experimento consiste em observar uma amostra (alturas das alunas-turma on-35), para analisar a média de altura das estudantes da turma on-35, com o objetivo de constatar se é diferente de 160 cm. 
Foram analisadas 30 alturas.


**4) Interprete os resultados - Preciso saber se rejeitou ou não a hipótese nula.**

**Formulando as Hipósteses:**

Hipótese Nula (H₀): A altura média da turma on-35 é igual a média nacional da população feminina brasileira de 160 cm.

Hipótese Alternativa (H₁) - A altura média da turma on-35 é diferente da média nacional da população feminina brasileira de 160 cm.


**Estatística z (z-statistic): 3.5569**

A estatística z de 3.5569 indica quantos desvios padrão a média da amostra (altura média da turma) está distante da média de referência (160 cm). 

Nesse caso, o valor positivo sugere que a média da turma está acima da média nacional.

**Interpretação do valor p**

Com base na estatística z de 3.5569, o valor p obtido é 0.0004, o que significa que a probabilidade de observar uma diferença tão grande ou maior, se a média da turma fosse 160 cm, é muito baixa (0.04%).

**Conclusão:**

O nível de significância utilizado é 0,05, o valor p obtido é 0.0004 é menor, que o nível de significância, sendo assim, *rejeitamos* a hipótese nula (H₀). 

Podemos afirmar que há evidências estatísticas de que a altura média da turma é significativamente diferente de 160 cm.

A probabilidade de que a média seja de fato 160 cm é muito baixa, e, com 95% de confiança, podemos dizer que a altura média da turma on-35 é diferente da média nacional.